src/routes/auth.js

```diff
@@ -71,6 +71,34 @@ userRouter.post('/login', loginValidation(), async (request, response) => {
     });
   }
 
+  // Change Password 
+  
+  router.post('/change_password', auth, (req, res,next) =>{
```

Comment:
ReferenceError: router is not defined
Change it to userRouter.

Reply:
Okay, on it.